crates/red_knot_python_semantic/src/semantic_index.rs

```diff
@@ -378,14 +379,16 @@ mod tests {
     impl UseDefMap<'_> {
         fn first_public_binding(&self, symbol: ScopedSymbolId) -> Option<Definition<'_>> {
             self.public_bindings(symbol)
-                .next()
                 .map(|constrained_binding| constrained_binding.binding)
+                .find(Option::is_some)
+                .unwrap()
```

Comment:
Nit: Can we use `find_map` here? Can we add a documentation why calling `unwrap` is safe?

Reply:
> Can we use `find_map` here?

Yes, thanks!



> Can we add a documentation why calling `unwrap` is safe?

It is *not* safe to call unwrap here. But note that this is only used in tests. But I changed it to `.expect(…)` now and added `#[track_caller]`.